pkg/config/config.go

```diff
@@ -604,8 +623,10 @@ func ReadRunModConfig(fileName string) (common.RunModConfig, error) {
 func ExtractReadRunModConfig(jsonData []byte) (common.RunModConfig, error) {
 	var runModConfig common.RunModConfig
 	if len(strings.TrimSpace(string(jsonData))) == 0 {
-		log.Error("ExtractReadRunModConfig: Could not parse runmod config as it is empty")
-		return runModConfig, fmt.Errorf("ExtractReadRunModConfig: No data to parse")
+		log.Infof("ExtractReadRunModConfig: Empty or no runmod config, using defaults from server.yaml")
+		runModConfig.PQSEnabled = IsPQSEnabled()
+		runModConfig.QueryTimeoutSecs = GetQueryTimeoutSecs()
```

Comment:
nit: now there's some duplication here with lines 637-638, and that duplication will grow as we add more things to the runmod config. It might be worth moving these two lines to a helper method to avoid the duplication